src/data/int/basic.lean

```diff
@@ -736,6 +739,13 @@ theorem neg_div_of_dvd : ∀ {a b : ℤ} (H : b ∣ a), -a / b = -(a / b)
 | ._ b ⟨c, rfl⟩ := if bz : b = 0 then by simp [bz] else
   by rw [neg_mul_eq_mul_neg, int.mul_div_cancel_left _ bz, int.mul_div_cancel_left _ bz]
 
+lemma sub_div_of_dvd {a b c : ℤ} (hcb : c ∣ b) : (a - b) / c = a / c - b / c :=
```

Comment:
Do you mind renaming this to `sub_div_of_dvd_right` and adding the `left` version?

Reply:
At least the obvious left version isn't true!
```lean 
lemma sub_div_of_dvd {a b c : ℤ} (hcb : c ∣ a) : (a - b) / c = a / c - b / c := by slim_check
===================
Found problems!

a := 0
b := 1
c := 2
issue: -1 = 0 does not hold
(5 shrinks)
```

I guess the version
```lean
lemma sub_div_of_dvd_sub {a b c : ℤ} (hcab : c ∣ (a - b)) : (a - b) / c = a / c - b / c :=
by rw [eq_sub_iff_add_eq, ← int.add_div_of_dvd_left hcab, sub_add_cancel]
```
is actually true, not sure how useful that is though?

Reply:
Oh, whoops. OK, we can just leave this as is then.

Reply:
Yeah I had exactly the same thought a few days ago and spent 5 mins trying to prove it 😬 I'll add this other version to the PR though, and maybe an add version of that too, why not lol